tuna/load_job.py

```diff
@@ -60,10 +60,6 @@ def parse_args():
       help='All configs with this tag will be added to the job table. \
                         By default adds jobs with no solver specified (all solvers).'
   )
-  config_filter.add_argument('--all_configs',
-                             dest='all_configs',
-                             action='store_true',
-                             help='Add all convolution jobs')
```

Comment:
I think it's better to keep all_configs, else we'll have to keep updating the tag in the sql db to get the same effect.